app/helpers/view_components/mentor/header.rb

```diff
@@ -47,43 +47,60 @@ def stats
         end
       end
 
-      def tabs
-        [
-          link_to(
-            Exercism::Routes.mentoring_inbox_path,
-            class: tab_class(:workspace)
-          ) do
-            graphical_icon(:overview) +
-              tag.span("Your Workspace") +
-              tag.span(number_with_delimiter(inbox_size), class: 'count')
-          end,
-
-          link_to(
-            Exercism::Routes.mentoring_queue_path,
-            class: tab_class(:queue)
-          ) do
-            graphical_icon(:queue) +
-              tag.span("Queue") +
-              tag.span(number_with_delimiter(queue_size), class: 'count')
-          end,
-
-          link_to(
-            Exercism::Routes.mentoring_testimonials_path,
-            class: tab_class(:testimonials)
-          ) do
-            graphical_icon(:testimonials) +
-              tag.span("Testimonials") +
-              tag.span(number_with_delimiter(num_testimonials), class: 'count')
-          end,
+      def tabs = [workspace_tab, queue_tab, testimonials_tab, automation_tab]
```

Comment:
The automation tab has logic in it to determine whether it is enabled or not. That led me to extracting the tabs into separate methods.